### PR TITLE
Ladybird: Restore ability to load emoji from Ladybird

### PR DESCRIPTION
**Pull request by trflynn89**
For example, this allows emoji to render on Strava.

Before:
before

After:
after

### CODE REVIEW

File: Ladybird/FontPlugin.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ FontPlugin::FontPlugin(bool is_layout_test_mode)
     Gfx::FontDatabase::set_default_font_query("Katica 10 400 0");
     Gfx::FontDatabase::set_fixed_width_font_query("Csilla 10 400 0");
 
-    Gfx::Emoji::set_emoji_lookup_path(String::formatted("{}/res/emoji", s_serenity_resource_root).release_value_but_fixme_should_propagate_errors());
+    Gfx::Emoji::set_emoji_lookup_path(String::formatted("{}/emoji", s_serenity_resource_root).release_value_but_fixme_should_propagate_errors());
```
Comment (nico) [inline Ladybird/FontPlugin.cpp:29]: Maybe good to be consistent with https://github.com/SerenityOS/serenity/pull/23016/files (?)

Comment (trflynn89) [reply Ladybird/FontPlugin.cpp:29]: I don't think that will work with installations, and that Ladybird has to load resources from `s_serenity_resource_root` (which changes depending on whether this is a dev build or a packaged installation)

Comment (trflynn89) [reply Ladybird/FontPlugin.cpp:29]: It's probably more correct for Ladybird to be using Core::Resource here actually, @ADKaster ?

Comment (nico) [reply Ladybird/FontPlugin.cpp:29]: Iirc MacPDF always sets s_serenity_source_root to the bundle dir, and the command line Lagom tools set it to the build dir instead. That could work for cocoa ladybird too?

Comment (trflynn89) [reply Ladybird/FontPlugin.cpp:29]: Looking at it closer, I think AK has us spiritually doing the same thing in Ladybird. We default to the bundle path on mac, starting from the exe path:
https://github.com/SerenityOS/serenity/blob/f58ca99a66db8f2661b3f3addd3453ab2619dcdd/Userland/Libraries/LibCore/System.cpp#L1807-L1810
And then from that path, we get:
https://github.com/SerenityOS/serenity/blob/f58ca99a66db8f2661b3f3addd3453ab2619dcdd/Ladybird/Utilities.cpp#L58

Dunno what Andrew's long term plan here is for multiple lagom apps, but sounds like he has some consolidation ideas floating around:
https://github.com/SerenityOS/serenity/pull/23304#discussion_r1498632046

Comment (trflynn89) [reply Ladybird/FontPlugin.cpp]: I pushed `Core::Resource` usage for now - that would have at least told us we forgot to bundle the emoji in #23304

Comment (ADKaster) [reply Ladybird/FontPlugin.cpp]: Yeah in theory we should be copying the files to Build/lagom/share/emoji as part of the build, and then any app that needs them can get them from that path (app_path/../share). For macOS app bundles, we'd need to copy the files into the app bundle under resources like I did for Ladybird in that PR. 

Still feels like a CFAppBundle Resource backend would be neat, then we could pretend that paths don't exist for these app bundles.

**Pull request merged**